File: examples/actions-matching/actions-matching.py
import json
import iam_floyd as statement

s = (
    # doc-start
    statement.Ec2() \
        .deny() \
        .all_matching_actions('/vpn/i')
    # doc-end
).to_json()

pretty = json.dumps(s, indent=4, sort_keys=True)
print(pretty)

File: examples/actions-matching/actions-matching.ts
import { deploy, out } from '../../helper/typescript/typescript_test';
import { Statement } from '../../lib';

function getStatement() {
  return (
    // doc-start
    new Statement.Ec2() //
      .deny()
      .allMatchingActions('/vpn/i')
    // doc-end
  );
}
const s = [getStatement()];
out(s);
deploy(s);
